### PR TITLE
added new ml, cs and math resources to README.md

### PR DESCRIPTION
**Pull request by marccodess**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -239,6 +239,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :bulb: [Open Source Society University - Path to a free self-taught education in Computer Science!](https://github.com/ossu/computer-science)
 - :books: [Teach Yourself Computer Science](https://teachyourselfcs.com/)
 - :books: [Introduction to Compilers and How they Work](https://www.edx.org/course/compilers)
+- :books: [Mathematics for Computer Science](https://people.csail.mit.edu/meyer/mcs.pdf)
```
Comment (shahednasser) [inline README.md:242]: I don't accept pdf sources as I can't assess its license so please remove this.

File: README.md [hunk 2]
```diff
@@ -353,6 +354,8 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :bulb: [Deep learning cheat sheet](https://stanford.edu/~shervine/teaching/cs-229/cheatsheet-deep-learning)
 - :books: [Natural Language Processing CS224n](http://web.stanford.edu/class/cs224n/)
 - :books: [IBM Deep Learning Course with certification](https://cognitiveclass.ai/courses/introduction-deep-learning) 
+- :books: [colah's blog](http://colah.github.io/)
```
Comment (shahednasser) [inline README.md:357]: This resource doesn't qualify please remove it.

File: README.md [hunk 3]
```diff
@@ -803,6 +807,8 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :books: [Deep Learning - University of buffalo](https://cedar.buffalo.edu/~srihari/CSE676/index.html)
 - :books: [Machine Learning Introduction with Python](https://www.dataquest.io/path/machine-learning-intro-with-python/)
 - :speaker: [Machine Learning Guide Podcast](https://ocdevel.com/mlg)
+- :books: [An Introduction to Statistical Learning with Applications in R](http://faculty.marshall.usc.edu/gareth-james/)
+- :books: [The Elements of Statistical Learning](https://hastie.su.domains/Papers/ESLII.pdf)
```
Comment (shahednasser) [inline README.md:811]: Please remove this

File: README.md [hunk 4]
```diff
@@ -803,6 +807,8 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :books: [Deep Learning - University of buffalo](https://cedar.buffalo.edu/~srihari/CSE676/index.html)
 - :books: [Machine Learning Introduction with Python](https://www.dataquest.io/path/machine-learning-intro-with-python/)
 - :speaker: [Machine Learning Guide Podcast](https://ocdevel.com/mlg)
+- :books: [An Introduction to Statistical Learning with Applications in R](http://faculty.marshall.usc.edu/gareth-james/)
```
Comment (shahednasser) [inline README.md:810]: This link seems to be linking to other websites and it doesn't contain anything specific or helpful. Either put the direct link to the resource or remove it.

File: README.md (hunk 2 above)
Comment (marccodess) [reply README.md]: Removed.

File: README.md [hunk 5]
```diff
@@ -496,6 +499,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :green_book: [Gauss & Determinant](https://cp-algorithms.com/linear_algebra/determinant-gauss.html)
 - :green_book: [Kraut & Determinant](https://cp-algorithms.com/linear_algebra/determinant-kraut.html)
 - :green_book: [Rank of a matrix](https://cp-algorithms.com/linear_algebra/rank-matrix.html)
+- :books: [Fundamentals of Linear Algebra](https://personal.math.ubc.ca/~carrell/NB.pdf)
```
[the remark this answers is not in the document]
Comment (marccodess) [reply README.md]: Removed PDF resource.

File: README.md (hunk 3 above)
Comment (marccodess) [reply README.md]: Removed PDF resource.

File: README.md (hunk 4 above)
Comment (marccodess) [reply README.md]: I have changed the link to a more preferred landing page for accessing this resource. There is a "Download the second edition" button at the top allowing the end user to access it. A must-read for anybody interested in Machine Learning. Thanks.

**Review by marccodess — COMMENTED** [review]
changes completed.

**Review by shahednasser — APPROVED** [review]

**Comment by shahednasser** [comment]
Thanks!

**Pull request merged**